packages/vx-geo/src/projections/Projection.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Group } from '@vx/group';
+import additionalProps from '../util/additionalProps';
+import { geoOrthographic, geoAlbers, geoMercator, geoPath } from 'd3-geo';
+
+// TODO: Implement all projections of d3-geo
+const projectionMapping = {
+  orthographic: geoOrthographic(),
```

Comment:
if there are multiple instances on a page of one type of projection, would singleton instances result in mixed or lingering config / props being set on them (e.g., if one set `clipAngle` and another didn't, would they both have the specified `clipAngle`)? 

you could get around it by making these factories as opposed to singletons?